src/pages/FileFields.tsx

```diff
@@ -31,38 +37,54 @@ const MainFileMetadata = styled.div`
 
 interface FileFieldsProps {
   uppy: Uppy;
+  formikProps: FormikProps<FormikValues>;
+  setAllChangesSaved: Dispatch<SetStateAction<boolean>>;
```

Comment:
hm, det har jeg ikke tenkt på... sendte en setState som prop selv ganske nylig, og da definerte jeg den bare som `(newValue: boolean) =>void` og det gikk helt fint. litt overrasket over at begge er lov, egentlig

Reply:
plukket det opp her : https://stackoverflow.com/questions/56028635/passing-usestate-as-props-in-typescript